composer/trainer/trainer.py

```diff
@@ -837,8 +863,7 @@ def _train_batch_inner(self, microbatches: Sequence[Batch]):
         current_batch_size = sum([self._train_data_spec.get_num_samples_in_batch(batch) for batch in microbatches])
 
         for microbatch_idx, state.batch in enumerate(microbatches):
-            state.batch_num_tokens = self._train_data_spec.get_num_tokens_in_batch(state.batch)
-            state.batch_num_samples = self._train_data_spec.get_num_samples_in_batch(state.batch)
```

Comment:
Cleared renaming this to `micro_batch_num_samples` with Ravi. It's not used anywhere and it makes the `state` object confusing, plus it messes w the decorator

Reply:
Small nit: let's use `microbatch` instead of `micro_batch` to be consistent with how it's named everywhere else in the codebase